server/src/main/resources/db/dev/R__Load_testing_data.sql

```diff
@@ -1090,6 +1090,10 @@ INSERT INTO feedback_requests
 VALUES
 ('ab2da7fc-fac2-11eb-9a03-0242ac130003', '59b790d2-fabc-11eb-9a03-0242ac130003', '2dee821c-de32-4d9c-9ecb-f73e5903d17a', '67dc3a3b-5bfa-4759-997a-fb6bac98dcf3' ,'18ef2032-c264-411e-a8e1-ddda9a714bae', '2021-08-01', '2021-08-05', null, 'pending');
 
+INSERT INTO feedback_requests
+(id, creator_id, requestee_id, recipient_id, template_id, send_date, due_date, submit_date, status) -- requestee: Zach Brown, recipient: Michael Kimberlin
+VALUES
+('ab2da7fc-fac2-11eb-9a03-0242ac130004', '59b790d2-fabc-11eb-9a03-0242ac130003', '43ee8e79-b33d-44cd-b23c-e183894ebfef', '6207b3fd-042d-49aa-9e28-dcc04f537c2d' ,'18ef2032-c264-411e-a8e1-ddda9a714bae', '2024-04-10', '2024-04-11', null, 'pending');
```

Comment:
Would it make sense to change this insertion so that the send date is always the current day? That way we always have a piece of test data that could be used to test the cron job.

Reply:
something like `CAST(GETDATE() As DATE)` for send_date...the other two dates can be null

Reply:
There is also (NOW())::date

Reply:
Won't this mean that an email will be sent from dev the day after we deploy to dev?

Reply:
yep...that's exactly what it would mean. and that's exactly what I want. :)

Reply:
Further, it means that we could force run right after deployment and use it as a smoke test.

Reply:
> There is also (NOW())::date

Mein SQL-foo ist scheiẞe.

Reply:
okey dokey